src/session/session_api.c

```diff
@@ -1566,6 +1566,7 @@ __session_upgrade_readonly(WT_SESSION *wt_session, const char *uri, const char *
 static int
 __session_verify(WT_SESSION *wt_session, const char *uri, const char *config)
 {
+    WT_CONFIG_ITEM cval;
```

Comment:
Don't we need to update the WiredTiger documentation for this ticket?

Reply:
Does this refer to command-line.dox? 

If so, I have updated it now 👍

Reply:
The documentation for verify() is here:
https://github.com/wiredtiger/wiredtiger/blob/pm-1521-durable-history/src/include/wiredtiger.in#L1684

This would need to be updated too.